deltachat-ffi/deltachat.h

```diff
@@ -4167,29 +4169,34 @@ int64_t          dc_lot_get_timestamp     (const dc_lot_t* lot);
  */
 #define DC_MSG_VIDEOCHAT_INVITATION 70
 
+/**
+ * @}
+ */
+
 
 /**
  * @defgroup DC_SOCKET DC_SOCKET
  *
  * These constants configure socket security.
+ * To set socket security,
+ * use dc_set_config() with the keys mail_security and/or send_security.
+ * If these flags are not specified, #DC_SOCKET_AUTO is used.
  *
  * @addtogroup DC_SOCKET
  * @{
  */
 
-/**
- * @}
- */
-
-
 /**
  * Choose socket security automatically.
+ * Automatic configuration is disabled if another @ref DC_SOCKET flag is specified
+ * or if server or port are specified.
  */
 #define DC_SOCKET_AUTO 0
 
 
 /**
  * Connect via SSL/TLS.
+ * If this flag is set, automatic configuration is skipped.
```

Comment:
Insetad of adding this line here and in `DC_SOCKET_PLAIN`, it should be removed from `DC_SOCKET_STARTTLS`. `DC_SOCKET_` constants are not flags, but an enumeration.

Reply:
i added a line to the common enum description.